lib/simctl.js

```diff
@@ -1,52 +1,47 @@
-import Q from 'q';
 import { exec } from 'teen_process';
-import { sleep, retry, nodeifyAll } from 'asyncbox';
+import { retryInterval } from 'asyncbox';
 import { getLogger } from 'appium-logger';
-import { mapify } from 'es6-mapify';
 
-const log = getLogger('node-simctl');
 
-function simExec (command:string, timeout:number, args:Array = []) {
+const log = getLogger('simctl');
 
-  args = ["simctl", command].concat(args);
+async function simExec (command:string, timeout:number, args:Array = []) {
+  // run a particular simctl command
+  args = ['simctl', command, ...args];
   log.info(`Executing: xcrun with args: ${args.join(' ')} and timeout: ${timeout}`);
 
-  let ret;
   try {
-    ret = exec("xcrun", args, {timeout});
+    return exec('xcrun', args, {timeout});
```

Comment:
going through some old PR notifications. don't we want to do `return await exec` here according to our new style? should probably get that written down somewhere too.